### PR TITLE
New package: trinsic.cli version 1.2.1

### PR DESCRIPTION
**Pull request by fundthmcalculus**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36769)

### CODE REVIEW

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+PackageIdentifier: trinsic.cli
+PackageVersion: 1.2.2
+PackageLocale: en-US
+Publisher: Trinsic (Streetcred ID)
+PublisherUrl: https://github.com/trinsic-id/sdk
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Trinsic
+PackageName: Trinsic Ecosystem CLI
```
Comment (ItzLevvie) [inline manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml:12]:
```suggestion
PackageName: trinsic
```

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=NVS1.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: trinsic.cli
+PackageVersion: 1.2.2
+PackageLocale: en-US
+Publisher: Trinsic (Streetcred ID)
```
Comment (ItzLevvie) [inline manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml:7]:
```suggestion
Publisher: Trinsic
```

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=NVS1.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: trinsic.cli
+PackageVersion: 1.2.2
```
Comment (ItzLevvie) [inline manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.2.1
```

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=NVS1.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: trinsic.cli
+PackageVersion: 1.2.2
```
Comment (ItzLevvie) [inline manifests/t/trinsic/cli/1.2.2/trinsic.cli.installer.yaml:5]:
```suggestion
PackageVersion: 1.2.1
```

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.3 $debug=NVS1.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: trinsic.cli
+PackageVersion: 1.2.2
```
Comment (ItzLevvie) [inline manifests/t/trinsic/cli/1.2.2/trinsic.cli.yaml:5]:
```suggestion
PackageVersion: 1.2.1
```

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+InstallerSwitches:
+  Silent: /qn
+  SilentWithProgress: /qb
+UpgradeBehavior: install
+Commands:
+- trinsic-cli
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/trinsic-id/sdk/releases/download/v1.2.0/trinsic-1.2.1-x86_64.msi
+  InstallerSha256: DE9C259D25DAFCB87BBA1D122666C0DA184BFEF3B62FCABC34E3207544F3F5ED
```
Comment (ItzLevvie) [inline manifests/t/trinsic/cli/1.2.2/trinsic.cli.installer.yaml:19]:
```suggestion
  InstallerSha256: DE9C259D25DAFCB87BBA1D122666C0DA184BFEF3B62FCABC34E3207544F3F5ED
  ProductCode: "{F9DBBC19-D931-4CD4-86C7-810BA89EF949}"
```

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml:12]: This must match Control Panel.

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.yaml (hunk 5 above)
Comment (ItzLevvie) [reply manifests/t/trinsic/cli/1.2.2/trinsic.cli.yaml:5]: This must match Control Panel.

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml:7]: This must match Control Panel.

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml:5]: This must match Control Panel.

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.installer.yaml (hunk 6 above)
Comment (ItzLevvie) [reply manifests/t/trinsic/cli/1.2.2/trinsic.cli.installer.yaml:19]: This must match Windows Registry.

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.installer.yaml (hunk 4 above)
Comment (ItzLevvie) [reply manifests/t/trinsic/cli/1.2.2/trinsic.cli.installer.yaml:5]: This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml (hunk 2 above)
Comment (fundthmcalculus) [reply manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml:7]: Will change control panel to match this, good to know, thanks!

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml (hunk 3 above)
Comment (fundthmcalculus) [reply manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml:5]: package is actually `1.2.2` due to release considerations.

File: manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml (hunk 1 above)
Comment (fundthmcalculus) [reply manifests/t/trinsic/cli/1.2.2/trinsic.cli.locale.en-US.yaml:12]: Will change control panel to match this, good to know, thanks!

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Missing changes:
1) https://github.com/microsoft/winget-pkgs/pull/36769/files#r761931719
2) https://github.com/microsoft/winget-pkgs/pull/36769/files#r761931684
3) https://github.com/microsoft/winget-pkgs/pull/36769/files#r761931560
4) https://github.com/microsoft/winget-pkgs/pull/36769/files#r761931438
5) https://github.com/microsoft/winget-pkgs/pull/36769/files#r761931776

Issue about having the version set to `1.2.2` is that, the package will always upgrade in a loop because `1.2.2` is larger than the Control Panel version of `1.2.1`; regardless if you're on the latest version or not. This wastes users' internet bandwidth.

PackageName and Publisher needs to match Control Panel because WinGet heavily relies on this. Incorrect PackageName / Publisher means that people may not be able to upgrade your application (ProductCode should bypass this), but it's still good practice to be consistent with Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Missing:
https://github.com/microsoft/winget-pkgs/pull/36769/files#r761931684

You will also need to update the version folder by pressing the `.` (dot) on your keyboard on this PR which will bring VS Code.

**Comment by fundthmcalculus** [comment]
Got confirmation from leadership that these proposed changes are fine. (thumbsup)

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**